nvdaHelper/vbufBackends/mshtml/node.h

```diff
@@ -32,12 +32,13 @@ class MshtmlVBufStorage_controlFieldNode_t : public VBufStorage_controlFieldNode
 	DWORD HTMLChangeSinkCookey;
 	std::wstring language;
 	VBufStorage_controlFieldNode_t*  ariaLiveNode;

```

Comment:
You can probably make this `MshtmlVBufStorage_controlFieldNode_t` instead. I can't see why it would be a specific type. Then casting wouldn't be necessary.